Lib/ctypes/_endian.py

```diff
@@ -3,34 +3,41 @@
 
 _array_type = type(Array)
 
+def _get_other_endian_attr():
+    """Return the appropriate other endian attribute based on system byte order."""
+    return "__ctype_be__" if sys.byteorder == "little" else "__ctype_le__"
+
 def _other_endian(typ):
-    """Return the type with the 'other' byte order.  Simple types like
+    """Return the type with the 'other' byte order. Simple types like
```

Comment:
Why you did this change?  PR, that just do a lot of code formatting changes - usually rejected by core developers.  See: https://devguide.python.org/getting-started/pull-request-lifecycle/#making-good-prs

```suggestion
    """Return the type with the 'other' byte order.  Simple types like
```